websites/V/Vizjer/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "334411435633541121",
+		"name": "meffiu"
+	},
+	"service": "Vizjer.pl",
```

Comment:
[Service name](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#service) should match the folder name.